src/window.rs

```diff
@@ -111,6 +112,8 @@ impl Builder {
             .with_title(self.title.clone());
 
         let context = glutin::ContextBuilder::new()
+            .with_gl_profile(GlProfile::Core)
+            .with_gl(GlRequest::Latest)
```

Comment:
I'd rather prefer to see a specific version here. @kvark what do you think?

Reply:
Some older GPUs only support GL 2.1, while some macs do *not* support older GL 2x. So if we are completely pedantic, we can't have a specific version requirement (even though, practically, GL 3.2 would fit most cases). So having `Latest` here is fine.